src/index.ts

```diff
@@ -110,11 +111,12 @@ const getAssetFromKV = async (event: FetchEvent, options?: Partial<Options>): Pr
   let shouldEdgeCache = false // false if storing in KV by raw file path i.e. no hash
   // check manifest for map from file path to hash
   if (typeof ASSET_MANIFEST !== 'undefined') {
-    if (JSON.parse(ASSET_MANIFEST)[pathKey]) {
-      pathKey = JSON.parse(ASSET_MANIFEST)[pathKey]
-      shouldEdgeCache = true // cache on edge if pathKey is a unique hash
+    if (ASSET_MANIFEST[pathKey]) {
+      pathKey = ASSET_MANIFEST[pathKey]
+      shouldEdgeCache = true // cache on edge if pathKey contains a unique hash
```

Comment:
nit: this comment is a bit misleading.
```suggestion
      // if path key is in asset manifest, we can assume it contains a content hash and can be cached
      shouldEdgeCache = true
```